huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_application_authorization_test.go

```diff
@@ -0,0 +1,212 @@
+package apig
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/apigw/dedicated/v2/appauths"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/common"
+)
+
+func getAppAuthFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := cfg.ApigV2Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating APIG v2 client: %s", err)
+	}
+
+	opts := appauths.ListOpts{
+		InstanceId: state.Primary.Attributes["instance_id"],
+		AppId:      state.Primary.Attributes["application_id"],
+	}
+	resp, err := appauths.ListAuthorized(client, opts)
+	if err != nil {
+		return nil, err
+	}
+	if len(resp) < 1 {
+		return nil, golangsdk.ErrDefault404{}
+	}
+	return resp, nil
+}
+
+func TestAccAppAuth_basic(t *testing.T) {
+	var (
+		authApis []appauths.ApiAuthInfo
+
+		rName      = "huaweicloud_apig_application_authorization.test"
+		rc         = acceptance.InitResourceCheck(rName, &authApis, getAppAuthFunc)
+		baseConfig = testAccAppAuth_base()
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppAuth_basic_step1(baseConfig),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+				),
+			},
+			{
+				Config: testAccAppAuth_basic_step2(baseConfig),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccAppAuthImportIdFunc(rName),
+			},
+		},
+	})
+}
+
+func testAccAppAuthImportIdFunc(rsName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[rsName]
+		if !ok {
+			return "", fmt.Errorf("resource (%s) not found: %s", rsName, rs)
+		}
+
+		instanceId := rs.Primary.Attributes["instance_id"]
+		resourceId := rs.Primary.ID
+		if instanceId == "" || resourceId == "" {
+			return "", fmt.Errorf("missing some attributes, want '<instance_id>/<id>' or "+
+				"'<application_id>/<envId>/<application_id>', but got '%s/%s'", instanceId, resourceId)
```

Comment:
```suggestion
		return "", fmt.Errorf("missing some attributes, want '<application_id>/<envId>/<application_id>', but got '%s/%s'", instanceId, resourceId")
```

Reply:
the <instance_id> is necessary, but the description actually is wrong.
s/applicaiton_id/instance_id